packages/cli/src/commands/setup/shadowenv/templates/rw.lisp.template

```diff
@@ -0,0 +1 @@
+(env/prepend-to-pathlist "PATH" (expand-path "./node_modules/.bin/"))
```

Comment:
Just noting that this adds everything in `node_modules/.bin`, which is a lot[^sn]. It's nice in that you can also run `rwfw`, but maybe we should just add `redwood` and `rw`? What do you think @thedavidprice?

[^sn]: ![image](https://user-images.githubusercontent.com/32992335/135935836-a55009e9-53e7-46bb-a339-ffdd2dfdc7a8.png)

Reply:
Ah, agreed that we should limit scope to `redwood` and `rw` and `rwfw`. Possible? 

If so, could include in final output something like "You are now aliasing `redwood`, `rw`, and `rwfw`. You can add other commands manually. See this doc ..." But up to you about this part.

Reply:
added via dee3159e31f807de7342fc794f6dfec11716b3c1